sktalk/corpus/conversation.py

```diff
@@ -1,17 +1,47 @@
+from typing import Dict
+from typing import List
+
+
 class Conversation:
     def __init__(
-        self, utterances: list["Utterance"], metadata: dict  # noqa: F821
+        self, utterances: List["Utterance"], metadata: Dict = None  # noqa: F821
     ) -> None:
+        """Representation of a transcribed conversation
+
+        Args:
+            utterances (list[Utterance]): A list of Utterance objects representing the utterances in the conversation.
+            metadata (dict, optional): Additional metadata associated with the conversation. Defaults to None.
+        """
         self._utterances = utterances
         self._metadata = metadata
 
     @property
     def utterances(self):
+        """
+        Get the list of utterances in the conversation.
+
+        Returns:
+            list[Utterance]: A list of Utterance objects representing the utterances in the conversation.
+        """
+
         return self._utterances
 
     @property
     def metadata(self):
+        """
+        Get the metadata associated with the conversation.
+
+        Returns:
+            dict: Additional metadata associated with the conversation.
+        """
         return self._metadata
 
     def get_utterance(self, index) -> "Utterance":  # noqa: F821
         pass
```

Comment:
Not changed here, but I suppose this should actually raise a `NotImplemented` exception:

```suggestion
        raise NotImplementedError
```